plutus-chain-index/src/Cardano/Index/Datum.hs

```diff
@@ -0,0 +1,95 @@
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE OverloadedStrings  #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+module Cardano.Index.Datum
```

Comment:
Why does this start with `Cardano.` and not with `Plutus` like all our packages?

Can this be `Plutus.Index.Datum`? even the executable is `plutus-indexer` (also `index`? `indexer`? pick one :-))

Reply:
My reasoning is that Indexers are generally more connected to `Cardano`, since that is what we are indexing, the blockchain. For example, `UTXOs` don't have any connection with `Plutus`.

Reply:
I agree with @raduom . However, the module name should reflect the package's name. For example, if we call the package `Marconi`, then the module should be `Marconi.Index.Datum`.